tests/cases/fourslash/goToDefinitionMetaProperty.ts

```diff
@@ -1,7 +1,12 @@
 /// <reference path='fourslash.ts' />
 
+// @Filename: /a.ts
 ////im/*1*/port.met/*2*/a;
-////function t() { n/*3*/ew.t/*4*/arget; }
+////function f() { n/*3*/ew.t/*4*/arget; }
+
+// @Filename: /b.ts
+////im/*5*/port.m;
+////class c { constructor() { n/*6*/ew.t/*7*/arget; } }

```

Comment:
Hm this is probably a seperate bug (I just wanted to check it didn't crash), but shouldn't goto-def on `new.target` at `7` here return `c`'s name's location?

Reply:
Oh, good point! Fixed in ec655c322adaadb6b17fd86fe6e1b07b005771b8